### ISSUE TITLE
ui crashes upon navigating to a video after download

### ISSUE THREAD

**Issue by almost-g** [TeamNewPipe/NewPipe#8321]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.23.0

### Steps to reproduce the bug

1. go to video
2. select download
3. go to Downloads 
4. start download
5. go to another video
6. crash

### Expected behavior

I would expect to navigate to a video, download it and then be able to navigate to another video.

### Actual behavior

I find a video to download, select download, go to Downloads and start downloading, when downloading completes I leave the Downloads menue and as soon as I navigate to a location where there are videos, CRASH.

## Exception
* __User Action:__ ui error
* __Request:__ ACRA report
* __Content Country:__ US
* __Content Language:__ en-US
* __App Language:__ en_US
* __Service:__ none
* __Version:__ 0.23.0
* __OS:__ Linux Android 5.1.1 - 22
<details><summary><b>Crash log </b></summary><p>

```
java.lang.OutOfMemoryError: Failed to allocate a 1048588 byte allocation with 906994 free bytes and 885KB until OOM
	at dalvik.system.VMRuntime.newNonMovableArray(Native Method)
	at android.graphics.BitmapFactory.nativeDecodeAsset(Native Method)
	at android.graphics.BitmapFactory.decodeStream(BitmapFactory.java:609)
	at android.graphics.BitmapFactory.decodeResourceStream(BitmapFactory.java:444)
	at android.graphics.drawable.Drawable.createFromResourceStream(Drawable.java:988)
	at android.content.res.Resources.loadDrawableForCookie(Resources.java:2474)
	at android.content.res.Resources.loadDrawable(Resources.java:2381)
	at android.content.res.Resources.getDrawable(Resources.java:787)
	at android.content.Context.getDrawable(Context.java:403)
	at androidx.core.content.ContextCompat$Api21Impl.getDrawable(ContextCompat.java:864)
	at androidx.core.content.ContextCompat.getDrawable(ContextCompat.java:483)
	at androidx.appcompat.widget.ResourceManagerInternal.getDrawable(ResourceManagerInternal.java:147)
	at androidx.appcompat.widget.ResourceManagerInternal.getDrawable(ResourceManagerInternal.java:135)
	at androidx.appcompat.content.res.AppCompatResources.getDrawable(AppCompatResources.java:104)
	at androidx.appcompat.widget.AppCompatImageHelper.setImageResource(AppCompatImageHelper.java:90)
	at androidx.appcompat.widget.AppCompatImageView.setImageResource(AppCompatImageView.java:98)
	at org.schabi.newpipe.local.subscription.item.FeedImportExportItem.addItemView(FeedImportExportItem.kt:80)
	at org.schabi.newpipe.local.subscription.item.FeedImportExportItem.setupImportFromItems(FeedImportExportItem.kt:104)
	at org.schabi.newpipe.local.subscription.item.FeedImportExportItem.bind(FeedImportExportItem.kt:43)
	at org.schabi.newpipe.local.subscription.item.FeedImportExportItem.bind(FeedImportExportItem.kt:21)
	at com.xwray.groupie.viewbinding.BindableItem.bind(BindableItem.java:72)
	at org.schabi.newpipe.local.subscription.item.FeedImportExportItem.bind(FeedImportExportItem.kt:37)
	at org.schabi.newpipe.local.subscription.item.FeedImportExportItem.bind(FeedImportExportItem.kt:21)
	at com.xwray.groupie.viewbinding.BindableItem.bind(BindableItem.java:50)
	at com.xwray.groupie.viewbinding.BindableItem.bind(BindableItem.java:23)
	at com.xwray.groupie.Item.bind(Item.java:49)
	at com.xwray.groupie.GroupAdapter.onBindViewHolder(GroupAdapter.java:233)
	at com.xwray.groupie.GroupAdapter.onBindViewHolder(GroupAdapter.java:20)
	at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:7337)
	at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:6194)
	at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6460)
	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6300)
	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6296)
	at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2330)
	at androidx.recyclerview.widget.GridLayoutManager.layoutChunk(GridLayoutManager.java:572)
	at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1591)
	at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:668)
	at androidx.recyclerview.widget.GridLayoutManager.onLayoutChildren(GridLayoutManager.java:170)
	at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4309)
	at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:4012)
	at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4578)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1076)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at androidx.viewpager.widget.ViewPager.onLayout(ViewPager.java:1775)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1076)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at androidx.coordinatorlayout.widget.CoordinatorLayout.layoutChild(CoordinatorLayout.java:1213)
	at androidx.coordinatorlayout.widget.CoordinatorLayout.onLayoutChild(CoordinatorLayout.java:899)
	at androidx.coordinatorlayout.widget.CoordinatorLayout.onLayout(CoordinatorLayout.java:919)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at androidx.drawerlayout.widget.DrawerLayout.onLayout(DrawerLayout.java:1263)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1703)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1557)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1466)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1703)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1557)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1466)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
	at android.view.View.layout(View.java:15693)
	at android.view.ViewGroup.layout(ViewGroup.java:5038)
	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2136)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1893)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1107)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5971)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:767)
	at android.view.Choreographer.doCallbacks(Choreographer.java:580)
	at android.view.Choreographer.doFrame(Choreographer.java:550)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:753)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5258)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:970)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:765)

```
</details>
<hr>

### Affected Android/Custom ROM version

5.1.1

### Affected device model

U704RC

**Comment by litetex** [comment]
Duplicate of e.g. #8307

TL;DR
Your device runs out of memory.
